wger/core/views/fitbit.py

```diff
@@ -0,0 +1,101 @@
+import requests
+import urllib
+import base64
+import os
+
+
+class FitBit:
+    """Class to handle all fitbit operation"""
+    # App settings from fitbit as regards the app
+    CLIENT_ID = os.getenv('CLIENT_ID')
```

Comment:
Let's handle cases where the `env` variable is not set. ie `CLIENT_ID = os.getenv('CLIENT_ID', 'DEFAULT_VALUE')`. Do this for the rest of the env variables.